content/en/examples/_index.html

```diff
@@ -0,0 +1,458 @@
+---
+title: "Fission Examples"
+linkTitle: "Examples"
+description: "Fission Examples - Sample applications from the community and the Fission team"
```

Comment:
We could use something like - `Examples of Fission serverless functions by Fission community and team` - for meta description. Include serverless functions as a the keyword. so that when users search for serverless functions examples, this page shows up in search.